### PR TITLE
Update install_overview.md

### PR DESCRIPTION
**Pull request by Rmalavally**

### CODE REVIEW

**Review by samjwu — APPROVED** [review]

**Review by samjwu — COMMENTED** [review]
Does the page at `release/licensing.md` (https://github.com/RadeonOpenCompute/ROCm/blob/develop/docs/release/licensing.md) have to use a different EULA for `rocm-llvm-alt`?

File: docs/deploy/linux/install_overview.md [hunk 1]
```diff
@@ -43,8 +43,9 @@ terms of this agreement, do not install, copy or use the AQL Profiler and/or the
 AOCC CPU Optimizations.
 ```
 
-For the rest of the ROCm packages, you can find the licensing information at the
-following location: `/opt/rocm/share/doc/<component-name>/`
+For the rest of the ROCm packages, you can find the licensing information at,
+
+https://www.amd.com/en/support/linux-eula
```
[the remark this answers is not in the document]
Comment (Rmalavally) [reply docs/deploy/linux/install_overview.md:48]: Freddy Paul approved this link and said it is more relevant than the general link at 

https://www.amd.com/en/support/amd-software-eula

I am ok with either.

**Comment by Rmalavally** [comment]
I think we use can use the general EULA link at https://www.amd.com/en/support/amd-software-eula on the llvm page.

**Comment by saadrahim** [comment]
> I think we use can use the general EULA link at https://www.amd.com/en/support/amd-software-eula on the llvm page.

rocm-llvm-alt is the only license information that needs to be fixed.

**Comment by samjwu** [comment]
Fixed in https://github.com/RadeonOpenCompute/ROCm/commit/ec8496041a35c08e6ad100e3e7a359de9573f704

**Pull request closed without merging**